packages/cli/src/commands/stream/commits.ts

```diff
@@ -1,22 +1,23 @@
-import { Command, type CommandFlags } from '../../command.js'
+import { Command, STREAM_ID_ARG, SYNC_ARG, SYNC_OPTIONS_MAP, type CommandFlags } from '../../command.js'
 
-export default class Commits extends Command<CommandFlags, { streamId: string }> {
+export default class Commits extends Command<CommandFlags, { streamId: string, sync: string }> {
   static description = 'list commits contained within a stream'
 
   static args = [
-    {
-      name: 'streamId',
-      description: 'ID of the stream',
-      required: true,
-    },
+    STREAM_ID_ARG,
+    SYNC_ARG,
   ]
 
   async run(): Promise<void> {
     this.spinner.start(`Loading stream ${this.args.streamId}...`)
     try {
-      const stream = await this.ceramic.loadStream(this.args.streamId)
+      const syncArg = SYNC_OPTIONS_MAP[this.args.sync]
+      const stream = await this.ceramic.loadStream(
+        this.args.streamId,
+        this.args.sync !== undefined ? { sync: syncArg} : undefined,
```

Comment:
Not an issue but it seems this could simply be set as `{ sync: this.args.sync  }` assuming the `loadStream()` method accepts `undefined` options values, or a `default` can be explicitly set as part of OCLIF's options?